gulpfile.js

```diff
@@ -14,31 +14,21 @@ const glob = require('glob');
 
 const env = process.env;
 
-gulp.task('webpack-dev', (cb) => {
+gulp.task('webpack-dev', async () => {
     preWebpack();
-    spawn(path.join(__dirname, './node_modules/.bin/webpack'), ['--mode', 'development'], { stdio: 'inherit', env }, cb);
+    return await spawn(path.join(__dirname, './node_modules/.bin/webpack'), ['--mode', 'development'], { stdio: 'inherit', env });
```

Comment:
As an alternative you can do this all in javascript:
```
const webpack = require('webpack');
const webpackConfig = require('./webpack.config');

webpackConfig.mode = mode;
return new Promise((resolve, reject) => webpack(webpackConfig, (err, stats) => {
    if (err) {
        reject(err);
    } else {
        console.log(stats.toString());
        resolve();
    }
}));
```

But I'm not sure how best to set `max-old-space-size` in that case

Reply:
Interesting, might consider that later.